dapps/profile.vue/src/components/verifications/notary/topic-display/topic-display.vue

```diff
@@ -29,9 +29,7 @@
   <evan-card class="mt-3" type="filled">
     <evan-loading v-if="loading"></evan-loading>
     <template v-else>
-      <svg width="60" height="56" viewBox="0 0 60 56" fill="none">
-        <path d="M59.3333 28L52.8267 20.5867L53.7333 10.7733L44.1067 8.58666L39.0667 0.106659L30 3.99999L20.9333 0.106659L15.8933 8.58666L6.26666 10.7467L7.17333 20.56L0.666664 28L7.17333 35.4133L6.26666 45.2533L15.8933 47.44L20.9333 55.92L30 52L39.0667 55.8933L44.1067 47.4133L53.7333 45.2267L52.8267 35.4133L59.3333 28ZM24.6667 41.3333L14 30.6667L17.76 26.9067L24.6667 33.7867L48.92 9.67999L53.7333 10.7733L53.5813 12.4187L24.6667 41.3333Z" fill="black"/>
-      </svg>
+      <i class="mdi mdi-check-decagram"></i>
```

Comment:
shall we use the new verification icon from the assets here and everywhere else instead of the mdi-icon?